### DISCUSSION TITLE
Passing additional meta-data to cloud-init

### DISCUSSION THREAD

**Discussion by 3rdcycle** [cockpit-project/cockpit-machines#717] [Q&A]
Besides user name and password, is it possible to pass additional meta-data to cloud-init when using cloud based images? I created a `config.iso` as a nocloud datasource but the cloud image based installation doesn't seem to let me attach an additional image on first launch.

**Comment by skobyda** [comment]
Hey.
No, root passwword, user login and user password are currently only configurable options. Can you tell me which specific metadata do you want to configure? Or do you want to upload the whole metadata file? 
If so, we can open a github issue for this and maybe it will get implemented in the future
